### ISSUE TITLE
ColoLight doesn't respect scene configuration

### ISSUE THREAD

**Issue by Simsz** [BazaJayGee66/homeassistant_cololight#43]
Not sure if this is my own improper configuration but when setting up scenes such as "Blue Mode" or "Green Mode" for my home assistant where all lights are, green or blue for example, the cololight doesn't retain this configuration and effect mode. Toggling the light effect modes manually seems to work just fine but when adding them to scenes it only retains the last known configuration. 

Am I missing something or does this just not work?

Sorry to open a whole issue on this -- just can't get it to work as I would imagine it would. Thank you!

**Comment by BazaJayGee66** [comment]
I've been able to reproduce the same issue, when trying to save an effect as part of a scene.
When activating the scene I get the following warning in the log:
```
[homeassistant.components.light.reproduce_state] Color mode hs specified but attribute hs_color missing for: light.test_light
```

Looks to work as expected when setting a colour, just not an effect.

Unsure what's causing the issue at the moment.

**Comment by BazaJayGee66** [comment]
This has been resolved in [v2.0.6](https://github.com/BazaJayGee66/homeassistant_cololight/releases/tag/v2.0.6).

Thanks for raising the issue.
